### ISSUE TITLE
ThemedPopupPanel with overflow doesn't scroll with the keyboard

### ISSUE THREAD

**Issue by adamconroy** [rstudio/rstudio#5592]
### System details

    RStudio Edition : RStudio Desktop Pro
    RStudio Version : 1.3.525.1
    OS Version         : Any
    R Version           : 3.6.1

Issue created in PR #5588 
### Steps to reproduce the problem
- Create a bunch of Terminal tabs
- Shrink the app window such that you get a scrollbar when opening the dropdown
- Keyboard navigate to a Terminal item and use the arrow keys to move focus down
- Observe that you scroll out of view

![3ZUhxXk](https://user-images.githubusercontent.com/136863/67243730-40867900-f426-11e9-849e-280d452aa883.png)

### Expected behavior
The panel view moves with the overflow scroll

### Issue resources and thoughts
I spent an hour looking into this. There are a few things are play here.
- Chrome/chromium doesn't properly support keyboard navigation with overflow:
https://groups.google.com/a/chromium.org/forum/#!msg/blink-dev/Fku6784p0wI/3MyOMTk7BwAJ
They've intended on shipping this for almost a year and still haven't

- We use tables (by virtue of GWT's defaults) liberally
Tables are notoriously quirky with focus and this is exacerbated by having multiple layers of nested tables in our popup panels.

- Keyboard navigation focus is not entirely in the user's control as we have a few instances where it's stolen or do not have tabindices set up, many of these will be smoothed out in 1.3 but it complicates this issue.

This blog post has some [admittedly aged] fixes that I was unable to make work in a short period of time but may lead us in a good direction: https://adrianroselli.com/2016/02/keyboard-and-overflow.html

We can always lean on JQuery-style manipulation as well if the browser won't do it for us though I'd want to test this pretty thoroughly, this SO post is a rough sketch of where we could start: https://stackoverflow.com/questions/20974241/table-scrolling-using-keyboard
